src/js/tabs/pid_tuning.js

```diff
@@ -779,6 +841,7 @@ TABS.pid_tuning.initialize = function (callback) {
             ADVANCED_TUNING.motorOutputLimit = parseInt($('.pid_tuning input[name="motorLimit"]').val());
             ADVANCED_TUNING.autoProfileCellCount = parseInt($('.pid_tuning input[name="cellCount"]').val());
             ADVANCED_TUNING.idleMinRpm = parseInt($('input[name="idleMinRpm-number"]').val());
+            RC_tuning.rates_type = $('select[id="ratesType"]').val();
```

Comment:
One important aspect of making this configurable in the UI is that this allows us to track how many users select each rates type by adding analytics tracking for this.
Can you please add something like this:

```
                let value = parseInt($('select[id="ratesType"]').val());

                let newValue = undefined;
                if (value !== RC_tuning.rates_type) {
                    newValue = $('select[id="ratesType"]').find('option:selected').text();
                }
                self.analyticsChanges['RatesType'] = newValue;

                RC_tuning.rates_type = value;
```

This will cause the selected rates type to be sent as analytics data.